pages/signup/signup.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/globals.scss';
+
+.container {
+	color: $textBlack;
+	background-color: #fff;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	padding: $space-xl;
+	margin-bottom: $space-xl;
+
+  h3 {
+    color: #183B56;
+    font-weight: bold;
+    font-family: $primaryHeaderFont;
+    padding-bottom: $space-sm;
+  }
+  p {
+    color: #8692A6;
+    font-family: $primaryTextFont;
+    font-size: 22px;
+  }
+  .information{
+    margin-top: 10vw;
```

Comment:
please use spacing vars for padding and margin

Reply:
✔️